packages/agreement-process/test/testSuspendAgreement.ts

```diff
@@ -69,182 +69,12 @@ export const testSuspendAgreement = (): ReturnType<typeof describe> =>
       vi.useRealTimers();
     });
 
-    it("should succeed when requester is Consumer or Producer, Consumer has all attributes satisfied, and the Agreement is in an suspendable state", async () => {
```

Comment:
No need to set valid attributes anymore to suspend the agreement by consumer or producer, see PR description